core/src/serve_repair.rs

```diff
@@ -143,8 +145,10 @@ impl RequestResponse for AncestorHashesRepairType {
 #[derive(Default)]
 pub struct ServeRepairStats {
     pub total_requests: usize,
-    pub dropped_requests: usize,
+    pub total_dropped_requests: usize,
+    pub total_dropped_response_packets: usize,
     pub total_response_packets: usize,
+    pub total_response_bytes: usize,
```

Comment:
not sure the extra `total_` prefix is adding much info here.

Reply:
was trying to align it with the existing `total_response_packets` and `total_requests`. Helps convey it's a count